app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    list_command = command.split()
+    source_file, destination_file = list_command[1], list_command[2]
```

Comment:
you must check that list_command has 3 arguments before this line. because code can be broken